src/bootstrap/src/core/builder/mod.rs

```diff
@@ -922,13 +922,13 @@ impl<'a> Builder<'a> {
                 test::Incremental,
                 test::Debuginfo,
                 test::UiFullDeps,
-                test::CodegenCranelift,
-                test::CodegenGCC,
                 test::Rustdoc,
                 test::CoverageRunRustdoc,
                 test::Pretty,
                 test::Crate,
                 test::CrateLibrustc,
+                test::CodegenCranelift,
+                test::CodegenGCC,
```

Comment:
Adding an explanatory comment would be great similar to the one at https://github.com/rust-lang/rust/blob/be32f13a44e5bc588ecb5289bfd6305d561fa26e/src/bootstrap/src/core/builder/mod.rs#L1029-L1035